specification/healthdataaiservices/resource-manager/readme.md

```diff
@@ -50,7 +59,6 @@ This is not used by Autorest itself.
 
 ```yaml $(swagger-to-sdk)
 swagger-to-sdk:
-  - repo: azure-sdk-for-python
```

Comment:
Python SDK could be generated from Typespec directly and no need to generate from swagger.